src/python/elements.cpp

```diff
@@ -331,10 +345,10 @@ void init_elements(py::module& m)
                  Aperture::Action const a = action == "transmit" ?
                      Aperture::Action::transmit :
                      Aperture::Action::absorb;
-                 return new Aperture(xmax, ymax, repeat_x, repeat_y, s, a, dx, dy, rotation_degree, name);
+                 return new Aperture(aperture_x, aperture_y, repeat_x, repeat_y, s, a, dx, dy, rotation_degree, name);
              }),
-             py::arg("xmax"),
-             py::arg("ymax"),
+             py::arg("aperture_x"),
+             py::arg("aperture_y"),
```

Comment:
As above: we could avoid this breaking change for the `Aperture` element itself.

Reply:
The part here might be ok, for python people might not use the named arguments yet that much.

Reply:
I'll keep this as breaking change for the Python API.